processing/src/main/java/org/apache/druid/audit/AuditManager.java

```diff
@@ -29,74 +28,76 @@
 
 public interface AuditManager
 {
-  /**
-   * This String is the default message stored instead of the actual audit payload if the audit payload size
-   * exceeded the maximum size limit configuration
-   */
-  String PAYLOAD_SKIP_MSG_FORMAT = "Payload was not stored as its size exceeds the limit [%d] configured by druid.audit.manager.maxPayloadSizeBytes";
 
   String X_DRUID_AUTHOR = "X-Druid-Author";
-
   String X_DRUID_COMMENT = "X-Druid-Comment";
 
   /**
-   * inserts an audit entry in the Audit Table
-   * @param key of the audit entry
-   * @param type of the audit entry
-   * @param auditInfo of the audit entry
-   * @param payload of the audit entry
-   * @param configSerde of the payload of the audit entry
+   * Value of header {@link #X_DRUID_AUTHOR} used by Druid services so that they
+   * can be distinguished from external requests.
+   */
+  String AUTHOR_DRUID_SYSTEM = "druid_system";
+
+  /**
+   * @return true if the audited event was initiated by the Druid system itself.
    */
-  <T> void doAudit(String key, String type, AuditInfo auditInfo, T payload, ConfigSerde<T> configSerde);
+  default boolean isSystemRequest(AuditInfo auditInfo)
+  {
+    return AUTHOR_DRUID_SYSTEM.equals(auditInfo.getAuthor());
```

Comment:
would this be valid for any auth extension?

Reply:
Yes, because we would explicitly set this as the header value when using internal clients.

Currently, we are setting it only in `OverlordClientImpl.runTask` but we can choose do set it in other audited endpoints too, or better yet just always set it by default in any service client. What do you think?

Reply:
So its possible for a user to bypass audit system if they set this header?